BE/OwnershipAndControl/Executives.rdf

```diff
@@ -463,10 +436,45 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>responsible party</rdfs:label>
-		<skos:definition>a person acting in a role that has some formal responsibility, such as a fiduciary responsibility, a signatory, an examiner or registrar, etc.</skos:definition>
-		<skos:editorialNote>The responsibility may be defined in terms of specific instruments such as laws, bylaws or contracts, or by some other means. These instruments would mandate certain obligations on the part of the party, for example statutory obligations.</skos:editorialNote>
+		<skos:definition>person acting in a role that has some formal responsibility, such as a fiduciary responsibility, a signatory, an examiner or registrar, etc.</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-be-oac-exec;Signatory">
+		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;AuthorizedParty"/>
+		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;LegallyDelegatedAuthority"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-law-lcap;hasCapacity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-fnd-law-lcap;SignatoryCapacity"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>signatory</rdfs:label>
+		<skos:definition>responsible party authorized to sign agreements on behalf of themselves, another person, or an organization</skos:definition>
 	</owl:Class>
 	
+	<owl:ObjectProperty rdf:about="&fibo-be-oac-exec;authorizes">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;actsOn"/>
```

Comment:
does not seem right

Reply:
It is, and works logically - actsOn is the proper relationship that is the parent in the lattice.  Hard to find a name that fits every case, but linguistically authorizing someone to do something has a primary role of 'actor'.